DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/HostApp/RhinoInstanceObjectsManager.cs

```diff
@@ -0,0 +1,258 @@
+using System.DoubleNumerics;
+using Rhino;
+using Rhino.DocObjects;
+using Rhino.Geometry;
+using Speckle.Connectors.Rhino7.Extensions;
+using Speckle.Connectors.Utils.Conversion;
+using Speckle.Connectors.Utils.Instances;
+using Speckle.Converters.Common;
+using Speckle.Core.Kits;
+using Speckle.Core.Logging;
+using Speckle.Core.Models;
+using Speckle.Core.Models.Instances;
+
+namespace Speckle.Connectors.Rhino7.HostApp;
+
+/// <summary>
+/// <inheritdoc/>
+///  Expects to be a scoped dependency per send or receive operation.
+/// </summary>
+public class RhinoInstanceObjectsManager : IInstanceObjectsManager<RhinoObject, List<string>>
```

Comment:
We are into Rhino now and my brain is melting and it's probably worth considering my ACAD comments first.

Also I wonder if these can be made to be more DRY though I _suspect_ not, maybe that's one for later IF

Reply:
They could be with an abstract class in between, but i'd settle with them as they are as "good enough" for now. We'd need to test this against other host apps to fully understand the patterns at play first (eg., tekla, revit)